scene/gui/tree.cpp

```diff
@@ -4839,15 +4839,18 @@ void Tree::_notification(int p_what) {
 	switch (p_what) {
 		case NOTIFICATION_EXIT_TREE:
 		case NOTIFICATION_ACCESSIBILITY_INVALIDATE: {
+#ifdef ACCESSKIT_ENABLED
 			if (root) {
 				_accessibility_clean_info(root);
 			}
 			for (ColumnInfo &col : columns) {
 				col.accessibility_col_element = RID();
 			}
 			accessibility_scroll_element = RID();
+#endif // ACCESSKIT_ENABLED
 		} break;
 
+#ifdef ACCESSKIT_ENABLED
```

Comment:
Doing it in `Tree`, but not in other controls, is pretty arbitrary. `NOTIFICATION_ACCESSIBILITY_UPDATE` won't be called if accessibility support is disabled, so this won't have any performance impacts.